client/app/cdn/dedicated/domain/statistics/cdn-dedicated-domain-statistics.controller.js

```diff
@@ -16,8 +44,13 @@ angular.module('App').controller('CdnDomainStatisticsCtrl', ($scope, $stateParam
     });
     $scope.series.push($translate.instant(`cdn_stats_legend_${$scope.model.dataType.toLowerCase()}_cdn`));
     $scope.series.push($translate.instant(`cdn_stats_legend_${$scope.model.dataType.toLowerCase()}_backend`));
-    $scope.data.push(_.map(_.get(data, 'cdn.values'), value => value.y));
-    $scope.data.push(_.map(_.get(data, 'backend.values'), value => value.y));
+    if ($scope.model.dataType === 'REQUEST') {
+      $scope.data.push(_.map(_.get(data, 'cdn.values'), value => value.y));
+      $scope.data.push(_.map(_.get(data, 'backend.values'), value => value.y));
+    } else if ($scope.model.dataType === 'BANDWIDTH' || $scope.model.dataType === 'QUOTA') {
+      $scope.data.push(_.map(_.get(data, 'cdn.values'), value => value.y / 1000000000)); // convert B to GB
```

Comment:
I think we have a dependency in `web` to convert numbers, you should use it

Reply:
is it available in dedicated?

Reply:
@radireddy For sure [`@ovh-ux/ng-ovh-web-universe-components`](https://github.com/ovh-ux/ng-ovh-web-universe-components) is available.

As you can see here:
- https://github.com/ovh-ux/ovh-manager-dedicated/blob/master/package.json#L65
- https://github.com/ovh-ux/ovh-manager-dedicated/blob/master/client/app/app.js#L13
- https://github.com/ovh-ux/ovh-manager-dedicated/blob/master/client/app/app.js#L52

In you case I think that @FredericEspiau refers to that following service:

- https://github.com/ovh-ux/ng-ovh-web-universe-components/blob/master/src/converter/converter.service.js

Hope it could helps you in a certain way.

Reply:
@antleblanc there is not utility function to convert Bytes to GB in converter.service.js. Should I add it and them use it in dedicated?

Reply:
Could be addressed in a separate pull request. Since this ticket is now in `quality check ok` then I will merge it.

Thanks:)